kb/current-run-id.md

```diff
@@ -0,0 +1,33 @@
+---
+slug: Non-determinism issues for Run Ids
+title: Non-determinism issues for Run Ids
+tags:
+  - kb-article
+date: 2022-10-25T00:00:00Z
+---
+
+## What is a Run Id
+
+A Run Id is a globally unique, platform-level identifier for a [Workflow Execution](/concepts/what-is-a-workflow-execution).
+
+## Overview
+
+The current Run Id is mutable and can change during a Workflow Retry. Because of that, you should not rely on storing the current Run Id, or using it for any logical choices, as a Workflow Retry changes the Run Id and can lead to non-determinism issues.
```

Comment:
```suggestion
The current [Run Id](/workflows#run-id) is mutable and can change during a Workflow Retry.
You should not rely on storing the current Run Id, or using it for any logical choices, because a Workflow Retry changes the Run Id and can lead to non-determinism issues.

<!-- truncate -->
```